spec/importers/data_transformers/csv_row_cleaner_spec.rb

```diff
@@ -34,7 +34,7 @@
         context 'date is not in the future or the far past' do
           let(:row) { CSV::Row.new([:event_date, :pizza_slices_count], ['2015-10-11', '3']) }
           it 'returns true' do
-            expect(row_cleaner.clean_date?).to eq true
+            expect(row_cleaner.send(:clean_date?)).to eq true
```

Comment:
I made all these methods private since only two are ever called.

Reply:
This is such a sneaky way to call a private method!! 👀